anmykh/homework/homework_6.py

```diff
@@ -0,0 +1,9 @@
+def max_function(*items, key=None):
+    minimum_value = 0
+    for item in list(items):
+        for number in item:
+            if number > minimum_value:
+                minimum_value = number
+            else:
+                pass
+        return minimum_value
```

Comment:
it will be wrong for 

def max_function("abc", "bcd", "klv")

Reply:
you either expect that there is 1 item - but collection of items, 
or more than 1 item

Reply:
also I see you do not use key param

Reply:
```
if len(items) == 1:
    items = items[0]
```